change/@ni-nimble-components-258183d7-7c0f-4208-978c-61b6089e7796.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

There are changes in the nimble package now
```suggestion
  "type": "patch",
```